spec/features/channel_filter_on_results_page_spec.rb

```diff
@@ -0,0 +1,178 @@
+RSpec.feature 'Consumer views channel filters on search results page' do
+  let(:landing_page) { LandingPage.new }
+  let(:results_page) { ResultsPage.new }
+
+  let!(:phone_advice)  { create(:other_advice_method, name: 'Advice by telephone', order: 1) }
+  let!(:online_advice) { create(:other_advice_method, name: 'Advice online (e.g. by video call / conference / email)', order: 2) }
+
+  scenario 'Consumer views current filters applied to their search' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_am_on_the_rad_landing_page
+      when_i_select_a_remote_advice_method
+      and_i_submit_the_remote_search_form
+      then_i_am_on_results_page
+      and_i_see_remote_advice_method_selected
+      and_only_remote_firms_are_displayed
+    end
+  end
+
+  scenario 'Consumer changes advice method from in person to remote' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_have_performed_an_in_person_search_from_the_landing_page
+      when_i_select_remote_advice
+      and_i_submit_the_sidebar_search_form
+      then_i_am_on_results_page
+      and_i_see_remote_advice_method_selected
+      and_only_remote_firms_are_displayed
+    end
+  end
+
+  scenario 'Consumer changes advice method from remote to in person' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_have_performed_a_remote_search_from_the_landing_page
+      when_i_select_face_to_face_advice
+      and_i_submit_the_sidebar_search_form
+      then_i_am_on_results_page
+      and_i_see_face_to_face_advice_method_selected
+      and_only_in_person_firms_are_displayed
+    end
+  end
+
+  scenario 'Consumer changes advice method to in person without providing postcode' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_have_performed_a_remote_search_from_the_landing_page
+      when_i_select_face_to_face_advice_without_providing_postcode
+      and_i_submit_the_sidebar_search_form
+      then_i_am_on_the_landing_page
+      and_a_validation_error_message_is_displayed
+    end
+  end
+
+  scenario 'Consumer changes advice method to remote without selecting remote advice type' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_have_performed_an_in_person_search_from_the_landing_page
+      when_i_select_remote_advice_without_providing_type
+      and_i_submit_the_sidebar_search_form
+      then_i_am_on_the_landing_page
+      and_a_validation_error_message_is_displayed
+    end
+  end
+
+
+  def given_some_firms_were_indexed
+    with_fresh_index! do
+      @phone_firm     = create(:firm, registered_name: 'The Willers Ltd',    in_person_advice_methods: [], other_advice_methods: [ phone_advice ])
```

Comment:
Can we make the formatting here consistent with the newer `face_to_face_*` and `phone_or_online_*` feature specs?